src/Umbraco.Examine.Lucene/UmbracoExamineIndex.cs

```diff
@@ -115,16 +115,32 @@ protected override void OnTransformingIndexValues(IndexingItemEventArgs e)
     {
         base.OnTransformingIndexValues(e);
 
+        // Performs special value transformations to all deriving indexes of this base class
+        // but the DeliveryApiContentIndex (they are not needed in this case).
+        // The alternative is to move the call to ApplySpecialIndexValueTransformations into a new base class
+        // that all implementors but the DeliveryApiContentIndex would inherit from but that would be breaking
+        // for any custom indexes deriving from UmbracoExamineIndex.
+        if (e.Index.Name != Constants.UmbracoIndexes.DeliveryApiContentIndexName)
```

Comment:
Wouldn't it be better to add an overridable property on this (base) class that by default is set to true and the DeliveryApiContentIndex overrides to false

Then check for this? This way we have no knowledge bleeding (base class knowing about the derived classes/properties/values)

This also gives the increased flexibility for implementors to turn off this base feature if they want to in their derived classes.

Reply:
Good point 💪 I pushed an update